lib/reports/usage.rb

```diff
@@ -27,7 +29,7 @@ def initialize(opts = {})
       if @include_child_usage
         children = (get_containers(runnable) - [runnable])
         children.each do |child|
-          col_defs << Reports::ColumnDefinition.new(:title => "#{child.name} (#{child.class}_#{child.id})\nAssessments Completed", :width => 4)
+          col_defs << Reports::ColumnDefinition.new(:title => @hide_names ? "#{child.class}_#{child.id}\nAssessments Completed" : "#{child.name} (#{child.class}_#{child.id})\nAssessments Completed", :width => 4)
```

Comment:
@dougmartin I think the `child.name` here is the name of the activity not the name of the student.